code/modules/power/smes.dm

```diff
@@ -154,7 +154,7 @@
 
 	//inputting
 	if(input_attempt && (!input_pulsed && !input_cut) && !grid_check)
-		target_load = clamp((capacity-charge)/SMESRATE, 0, input_level)	// Amount we will request from the powernet.
+		target_load = CLAMP((capacity-charge)/SMESRATE, 0, input_level)	// Amount we will request from the powernet.

```

Comment:
Pretty sure I merged that fix?

Reply:
I had to put that fix for me since I still use 512.